glusterd2/middleware/auth.go

```diff
@@ -36,18 +38,18 @@ func Auth(next http.Handler) http.Handler {
 			next.ServeHTTP(w, r)
 			return
 		}
-
+		ctx := r.Context()
 		// Verify if Authorization header exists or not
 		authHeader := strings.TrimSpace(r.Header.Get("Authorization"))
 		if authHeader == "" {
-			http.Error(w, "Authorization header is required", http.StatusUnauthorized)
+			restutils.SendHTTPError(ctx, w, http.StatusUnauthorized, errors.New("Authorization header is required"))
 			return
 		}
 
 		// Verify the Authorization header format "Bearer <TOKEN>"
 		authHeaderParts := strings.Split(authHeader, " ")
 		if len(authHeaderParts) != 2 || strings.ToLower(authHeaderParts[0]) != "bearer" {
-			http.Error(w, "Authorization header format must be Bearer <TOKEN>", http.StatusUnauthorized)
+			restutils.SendHTTPError(ctx, w, http.StatusUnauthorized, errors.New("Authorization header format must be Bearer <TOKEN>"))
```

Comment:
`Authorization header must be of the format - Bearer <TOKEN>`

Reply:
done